### ISSUE TITLE
Add docs to create a only HDD node

### ISSUE THREAD

**Issue by A-Harby** [threefoldtech/info_grid#595]
Update the docs with steps to create a node with no SSD (only HDD).

It is very important to mention Bois step: https://github.com/threefoldtech/zos/issues/2367#issuecomment-2208783018

https://github.com/threefoldtech/zos/issues/2258

**Comment by Mik-TF** [comment]
@A-Harby can I get more info here?

So in short, you just set up a 3Node without SSD, just HDD, and you make sure that the BIOS has AHCI set for SATA?

Is that all? Thanks. Will make the docs for the manual once this is clear.

**Comment by A-Harby** [comment]
> @A-Harby can I get more info here?
> 
> So in short, you just set up a 3Node without SSD, just HDD, and you make sure that the BIOS has AHCI set for SATA?
> 
> Is that all? Thanks. Will make the docs for the manual once this is clear.

Something that I struggled with for a while was how to specify a certain flag for the image, and I was able to do it with these steps to download the image with the correct/wanted flag:
1. Navigate to the Zero-OS Bootstrap [site](https://bootstrap.grid.tf/).
2. Select `Expert mode` from the navbar at the top.
3. Fill in all the needed inputs (Farm ID, network, kernel, and format).
4. In the `Extra kernel arguments` input field, type your specified flag (for example, here for only the HDD node, I used the flag `missing-ssd`).
5. Generate and download the image.


Other information should be in the issue and the other related issues.
